frontend/integration-tests/views/monitoring.view.ts

```diff
@@ -65,7 +65,8 @@ export const getFirstRowAsText = () => {
 };
 
 export const saveAsDefault = firstElementByTestID('save-as-default');
-
+export const sendResolvedAlerts = firstElementByTestID('send-resolved-alerts');
+export const showAdvConfiguration = $('button.pf-c-expandable__toggle');
```

Comment:
nit: Let's spell out the word. It's already a long name and editors have autocomplete.

```suggestion
export const showAdvancedConfiguration = $('button.pf-c-expandable__toggle');
```